### ISSUE TITLE
ValueError '/' is not in list.

### ISSUE THREAD

**Issue by njrichardson** [jim-schwoebel/download_audioset#4]
Traceback (most recent call last):
  File "as_download.py", line 160, in <module>
    clabels=convertlabels(ylabels[i],labels,textlabels)
  File "as_download.py", line 91, in convertlabels
    index=labels.index(sortlist[i])
ValueError: '/' is not in list

I am running into this error when I try and run the script. Im am on mac osx 10.15.4 and have all the dependancies installed. Any help with this would be be greatly appreciated.

**Comment by Irving-ren** [comment]
This error occurred because the ylabels[i] actually was saving the positive_labels, which is the same type as "/m/01g50p". It's easy to figure that out if you were debugging it. Anyway, the parameter of convertlabels function should be changed. Also, It will work properly if you add the function that could segment the clables variable into a certain format. 

Irving

**Comment by newbeepi** [comment]
> 
> 
> Traceback (most recent call last):
> File "as_download.py", line 160, in
> clabels=convertlabels(ylabels[i],labels,textlabels)
> File "as_download.py", line 91, in convertlabels
> index=labels.index(sortlist[i])
> ValueError: '/' is not in list
> 
> I am running into this error when I try and run the script. Im am on mac osx 10.15.4 and have all the dependancies installed. Any help with this would be be greatly appreciated.


2 codes have mistakes 
it  helped me:

1.  use as_download_unbalanced.py (just rename in 133 line)
2.  102 line change to os.system("youtube-dl -f bestaudio[ext=m4a] %s"%(link))
3.  remove pafy and shutil

**Comment by lsh3163** [comment]
This code is working for balanced.csv
```python
def convertlabels(sortlist,labels,textlabels):
    
    clabels=list()
    sortlist = sortlist.split(",") # What I add
    for i in range(len(sortlist)):
        #find index in list corresponding
        index=labels.index(sortlist[i])
        clabel=textlabels[index]
        #pull out converted label
        clabels.append(clabel)
    return clabels 
```

**Comment by jim-schwoebel** [comment]
Sorry for not maintaining this open source repo - it was rather sloppily constructed - and thank you @lsh3163 for suggesting the improvement. Fixed in 1af21f21c0633d72140d970f750d841a027c8942
